field.go

```diff
@@ -35,8 +35,17 @@ type Field = zapcore.Field
 var (
 	_minTimeInt64 = time.Unix(0, math.MinInt64)
 	_maxTimeInt64 = time.Unix(0, math.MaxInt64)
+
+	DebugLevelField = zapcore.DebugLevel
```

Comment:
needed to duplicate this in order to be able to pass the address of the `DebugLevel` for assignment (https://github.com/uber-go/zap/pull/1251/files#diff-b494c03820a49f9efc491691ed31a2acc16d4cebf39d679ffd74bf97bbee968bR45)

`DebugLevel` doesn't work because https://github.com/uber-go/zap/blob/master/level.go#L32 already uses that name in the same package.